numpy/core/tests/test_umath.py

```diff
@@ -986,17 +996,20 @@ def test_exp_values(self):
             yf = np.array(y, dtype=dt)
             assert_equal(np.exp(yf), xf)
 
-        with np.errstate(over='raise'):
-            assert_raises(FloatingPointError, np.exp, np.float32(100.))
-            assert_raises(FloatingPointError, np.exp, np.float32(1E19))
-            assert_raises(FloatingPointError, np.exp, np.float64(800.))
-            assert_raises(FloatingPointError, np.exp, np.float64(1E19))
-
-        with np.errstate(under='raise'):
-            assert_raises(FloatingPointError, np.exp, np.float32(-1000.))
-            assert_raises(FloatingPointError, np.exp, np.float32(-1E19))
-            assert_raises(FloatingPointError, np.exp, np.float64(-1000.))
-            assert_raises(FloatingPointError, np.exp, np.float64(-1E19))
+        # Older version of glibc may not raise the correct FP exceptions
+        # See: https://github.com/numpy/numpy/issues/19192
+        if (get_glibc_version() >= 2.17):
+            with np.errstate(over='raise'):
+                assert_raises(FloatingPointError, np.exp, np.float32(100.))
+                assert_raises(FloatingPointError, np.exp, np.float32(1E19))
+                assert_raises(FloatingPointError, np.exp, np.float64(800.))
+                assert_raises(FloatingPointError, np.exp, np.float64(1E19))
+
+            with np.errstate(under='raise'):
+                assert_raises(FloatingPointError, np.exp, np.float32(-1000.))
+                assert_raises(FloatingPointError, np.exp, np.float32(-1E19))
+                assert_raises(FloatingPointError, np.exp, np.float64(-1000.))
+                assert_raises(FloatingPointError, np.exp, np.float64(-1E19))
```

Comment:
It would make more sense to put these tests in a separate `def test_exceptions` test and use a `skipif` decorator.

Reply:
Fixed it.